OOInteraction/src/OOInteractionPlugin.cpp

```diff
@@ -49,6 +49,7 @@
 #include "commands/CCreateMethod.h"
 #include "commands/CSceneHandlerItemTest.h"
 #include "commands/CDoxygen.h"
+#include "commands/AddNodeCommand.h"
```

Comment:
Please change the AddNodeCommand class to follow the right conventions CXXX.

Also AddNode is too unspecific. Use CAddNodeToView